dhnx/optimization_add_components.py

```diff
@@ -357,8 +357,8 @@ def add_heatpipes(it, labels, gd, q, b_in, b_out, nodes):
         Table of *Heatpipeline* attributes of the district heating grid
     labels : dict
         Dictonary containing specifications for label-tuple
-    gd : dict
-        Settings of the investment optimisation of the ThermalNetwork
+    bidi : bool
```

Comment:
```suggestion
    bidirectional : bool
```

Reply:
A bit longer, but easier to understand. Could consider giving more telling names to 'q' and 'b_in' (bus_in, e.g.)